.lagoon.yml

```diff
@@ -4,13 +4,36 @@ project: dpl-cms-core
 
 ssh: 20.238.147.183:22
 api: https://api.lagoon.dplplat01.dpl.reload.dk/graphql
+environment_variables:
+  git_sha: 'true'
 
 tasks:
   post-rollout:
+    - run:
+        name: Create new GH deployment
+        command: |
+          GH_DEPLOYMENTS=$(curl -H "Authorization: Bearer $GH_DEPLOYMENT_TOKEN" \
+          "https://api.github.com/repos/danskernesdigitalebibliotek/dpl-cms/deployments?ref=$LAGOON_PR_HEAD_BRANCH&environment=$LAGOON_ENVIRONMENT")
+          GH_DEPLOYMENT_ID=$(echo "$GH_DEPLOYMENTS" | grep '"id":' | head -n 1 | sed 's/[^0-9]*\([0-9]*\).*/\1/')
```

Comment:
Could the deployment request be done with `gh` instead of a curl request?
I think the "ttps://api.github.com/repos/danskernesdigitalebibliotek/dpl-cms/deployments, or at least the GH org/repo should be defined in a repository variable

Reply:
Yes it can, but, then we need to have the `gh` command on lagoon, which we don't
I'd prefer not to introduce a new package :)

Reply:
It would have been much easier with `gh` as I'd also be able to use jq, but oh well :)